### PR TITLE
Integration & Acceptance tests

### PR DESCRIPTION
**Pull request by chingor13**
Adds a new cloudbuild pipeline for the e2e/integration tests. We run the pipeline against the images we just built (including the builder pipeline) after building the images.

Added a new `scripts/run_acceptance_tests.sh` which runs the same integration tests but with the production images and the production build pipeline.

### CODE REVIEW

File: testapps/build_pipeline/app.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+runtime: php
+env: flex
+
+runtime_config:
+  document_root: web
+
+env_variables:
+  WHITELIST_FUNCTIONS: phpinfo,exec
```
Comment (tmatsuo) [inline testapps/build_pipeline/app.yaml]: Can it be in the runtime_config?

```
runtime_config:
  document_root: web
  whiltelist_functions: phpinfo,exec
```

File: testapps/build_pipeline/nginx-app.conf [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2015 Google Inc.
```
Comment (tmatsuo) [inline testapps/build_pipeline/nginx-app.conf]: I think we don't need this file

Comment (tmatsuo) [inline testapps/build_pipeline/nginx-app.conf]: I don't think we need this file

File: testapps/build_pipeline/app.yaml (hunk 1 above)
Comment (chingor13) [inline testapps/build_pipeline/app.yaml]: I think we should move it here once we switch to the builders. All the other examples use env variables for now.

File: testapps/build_pipeline/nginx-app.conf (hunk 2 above)
Comment (chingor13) [inline testapps/build_pipeline/nginx-app.conf]: Ok, I'll remove it. This was copied from the e2e test.

File: testapps/build_pipeline/app.yaml (hunk 1 above)
Comment (chingor13) [inline testapps/build_pipeline/app.yaml]: Ah, I see. The runtime builder already implemented this, so we're making sure that we're using the builder.

**Pull request merged**